SoftLayer/managers/cdn.py

```diff
@@ -140,7 +140,7 @@ def purge_content(self, unique_id, path):
         :param str path: A string of url or path that should be purged.
         :returns: A Container_Network_CdnMarketplace_Configuration_Cache_Purge array object
         """
-
+        path = "/{}".format(path)
```

Comment:
I'd prefer we start using [Pyhton F strings](https://peps.python.org/pep-0498/) instead of .format
```suggestion
        path = f"/{path}"
```

Reply:
Done